README.md

```diff
@@ -62,6 +62,47 @@ npm run dev [component-name(s)]
 
 ### Testing
 
+#### ✨ New tests ([Web Test Runner](https://modern-web.dev/docs/test-runner/overview/))
+
+```shell
+# Run all tests in watch mode.
+npm run test:watch
+
+# Run a single test in watch mode.
+npm run test:watch --element="pfe-select"
+# Or multiple:
+npm run test:watch --element="{pfe-select,pfe-card}"
```

Comment:
I ended up having to include the `{}` in the parameter. 🙃 

TLDR: `{pfe-select}` won't work. It needs at minimum two strings for the {} to work.